{"uid":"3e5a4040588f6a16","name":"test_logout_user","fullName":"tests.test_user.TestAuthUser#test_logout_user","historyId":"1d2e077bd5984a013d21d49fd7455ddc","time":{"start":1752011317435,"stop":1752011317435,"duration":0},"description":"Can logout user","descriptionHtml":"<p>Can logout user</p>\n","status":"failed","statusMessage":"AssertionError: 409, {\"success\":false,\"status\":409,\"message\":\"An account already exists with the same email address\"}","statusTrace":"request = <SubRequest 'register_user' for <Function test_logout_user>>\n\n    @pytest.fixture()\n    def register_user(request):\n>       user, register_data = api_user.create_new_user()\n\nconftest.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.users.users_api.UserAPI object at 0x7f431ea46f60>\n\n    @allure.step('Register a new user')\n    def create_new_user(self):\n        payload = self.payloads.create_user()\n        response = MyRequests.post(\n            url=self.endpoints.create_user,\n            json=payload.model_dump()\n        )\n>       assert response.status_code == 201, f'{response.status_code}, {response.content.decode(\"utf-8\")}'\nE       AssertionError: 409, {\"success\":false,\"status\":409,\"message\":\"An account already exists with the same email address\"}\n\nservices/users/users_api.py:25: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1752011306869,"stop":1752011306905,"duration":36},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"attachmentStep":false},{"name":"_xunit_setup_method_fixture_TestUser.TestAuthUser","time":{"start":1752011317436,"stop":1752011317436,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"attachmentStep":false},{"name":"register_user","time":{"start":1752011317436,"stop":1752011317543,"duration":107},"status":"failed","statusMessage":"AssertionError: 409, {\"success\":false,\"status\":409,\"message\":\"An account already exists with the same email address\"}\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/test_project/conftest.py\", line 19, in register_user\n    user, register_data = api_user.create_new_user()\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/test_project/services/users/users_api.py\", line 25, in create_new_user\n    assert response.status_code == 201, f'{response.status_code}, {response.content.decode(\"utf-8\")}'\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Register a new user","time":{"start":1752011317436,"stop":1752011317543,"duration":107},"status":"failed","statusMessage":"AssertionError: 409, {\"success\":false,\"status\":409,\"message\":\"An account already exists with the same email address\"}\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/test_project/services/users/users_api.py\", line 25, in create_new_user\n    assert response.status_code == 201, f'{response.status_code}, {response.content.decode(\"utf-8\")}'\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"POST request to URL \"https://practice.expandtesting.com/notes/api/users/register\"","time":{"start":1752011317437,"stop":1752011317543,"duration":106},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"attachmentStep":false}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":0,"attachmentStep":false}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"attachmentStep":false}],"afterStages":[{"name":"_xunit_setup_method_fixture_TestUser.TestAuthUser::0","time":{"start":1752011317587,"stop":1752011317587,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"attachmentStep":false}],"labels":[{"name":"feature","value":"Login/logout user"},{"name":"epic","value":"Users"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_user"},{"name":"subSuite","value":"TestUser"},{"name":"host","value":"4c2e67bdc6b5"},{"name":"thread","value":"14-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"3e5a4040588f6a16.json","parameterValues":[]}